Drop after lang/rust >= 1.80 update

error[E0658]: use of unstable library feature 'lazy_cell'
 --> ../libcosmic-b40839638ab0e1d96de3f817eded647e6952db40/src/config/mod.rs:9:17
  |
9 | use std::sync::{LazyLock, Mutex};
  |                 ^^^^^^^^
  |
  = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

error[E0658]: use of unstable library feature 'lazy_cell'
  --> ../libcosmic-b40839638ab0e1d96de3f817eded647e6952db40/src/config/mod.rs:14:23
   |
14 | pub static COSMIC_TK: LazyLock<Mutex<CosmicTk>> = LazyLock::new(|| {
   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

error[E0658]: use of unstable library feature 'lazy_cell'
  --> ../libcosmic-b40839638ab0e1d96de3f817eded647e6952db40/src/config/mod.rs:14:51
   |
14 | pub static COSMIC_TK: LazyLock<Mutex<CosmicTk>> = LazyLock::new(|| {
   |                                                   ^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

error[E0658]: use of unstable library feature 'lazy_cell'
  --> ../libcosmic-b40839638ab0e1d96de3f817eded647e6952db40/src/config/mod.rs:14:51
   |
14 | pub static COSMIC_TK: LazyLock<Mutex<CosmicTk>> = LazyLock::new(|| {
   |                                                   ^^^^^^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

--- ../libcosmic-b40839638ab0e1d96de3f817eded647e6952db40/src/lib.rs.orig	2024-08-02 18:27:08 UTC
+++ ../libcosmic-b40839638ab0e1d96de3f817eded647e6952db40/src/lib.rs
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MPL-2.0
 
 #![allow(clippy::module_name_repetitions)]
+#![feature(lazy_cell)]
 
 #[cfg(all(feature = "wayland", feature = "winit"))]
 compile_error!("cannot use `wayland` feature with `winit`");
